doc-templates/Flex.md

```diff
@@ -10,8 +10,8 @@
 
 To enable this turn on `FlexRouting` as a feature in `otp-config.json`.
 
-The GTFS feeds should conform to the 
-[GTFS-Flex v2 draft PR](https://github.com/google/transit/pull/388)
+The GTFS feeds must conform to the final, approved version of the draft which has been 
+merged into the [mainline specification](https://gtfs.org/schedule/reference/) in March 2024.
```

Comment:
Should we clarify which experimental features are supported as this now introduces two which are not included in the reference.

Reply:
We discussed this in the dev meeting. We should specify that we support these experimental fields but we will change the implementation in otp if the spec changes so people don't rely on these too heavily.

Reply:
90148e089134c2289f1d6310bb50cf1645f0d000